components/Breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -12,6 +13,16 @@ interface BaseBreadcrumbLinkProps {
   to?: string
 }
 
+const LinkWrapper = ({ to, isCurrent, children }) => {
+  return isCurrent ? children : (
+    <NextLink href={to} passHref>
+      <Link>
+        {children}
+      </Link>
+    </NextLink>
+  )
+}
```

Comment:
I'd guess that this would be valuable as a standalone reusable component?

Reply:
I think yes